code/utils.q

```diff
@@ -559,24 +577,24 @@ utils.getRelevantNames:{[nameInfo;namedPathStem]
     '"savedModelName must be a string"
     ];
   if[0=count relevantModels;
-    '"No files matching the user provided savedModelName were found for deletion"
+  '"No files matching the user provided savedModelName were found for deletion"
```

Comment:
Should be 2 spaces indented for if statement

Reply:
this was done but needed to break long line < 80 characters